utilities/convert_frosst/smirffishFrcmod.parm99Frosst.txt

```diff
@@ -135,18 +135,18 @@ ANGL
 [*:1]~[#6X4:2]-[*:3]      50.  109.5   parm99 generic Csp3
 [#1:1]-[#6X4:2]-[#1:3]    35.  109.5   parm99 generic Csp3
 # Csp3 in 3-membered ring
-[*r3:1]1~@[*r3:2]~@[*r3:3]1 350.  60.     Frosst generic any atom within the 3-membered ring
-[*r3:1]~@[*r3:2]~!@[*:3] 100.  118.    Frosst generic 3ringAtom-3ringAtom-substit
-[*:1]~!@[*r3:2]~!@[*:3]  100.  113.    Frosst generic substit-3ringAtom-nonHsubstit
-[#1:1]-[*r3:2]~!@[*:3]    50.  113.    Frosst generic substit-3ringAtom-Hsubstit
+[*;r3:1]1~;@[*;r3:2]~;@[*;r3:3]1 350.  60.     Frosst generic any atom within the 3-membered ring
+[*;r3:1]~;@[*;r3:2]~;!@[*:3] 100.  118.    Frosst generic 3ringAtom-3ringAtom-substit
+[*:1]~;!@[*;r3:2]~;!@[*:3]  100.  113.    Frosst generic substit-3ringAtom-nonHsubstit
```

Comment:
So the record is here, these were valid before, it is a personal preference that makes the parsed smirks into environment objects make more sense.